cmd/backup-operator/README.md

```diff
@@ -0,0 +1,80 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+Try out etcd backup operator by running it on Kubernetes and then create a `EtcdBackup` Custom Resource which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `EtcdBackup` Custom Resource, retrieves etcd snapshot, and then saves it as a backup to S3.
+>Note: The demo uses the `default` namespace.
+
+Prerequisites: 
+* access to a Kubernetes environment. See TODO.
+* a running etcd cluster named `example-etcd-cluster`. See [Instruction][etcd_cluster_deploy].
+
+
+Once `example-etcd-cluster` is running, let's create a backup for `example-etcd-cluster` using etcd backup operator. 
+
+First, deploy an etcd backup operator:
+> Note: etcd backup operator also creates `EtcdBackup` CRD when starting.
+
+```sh
+$ kubectl create -f example/etcd-backup-operator/deployment.yaml
+$ kubectl get pod
+NAME                                    READY     STATUS    RESTARTS   AGE
+etcd-backup-operator-1102130733-hhgt7   1/1       Running   0          3s
+...
+```
+
+Then, create an Kubernetes secret that contains aws config/credential; etcd operator uses the secret to gain access to S3 in order to save the etcd snapshot.
```

Comment:
I found the whole thing a little unstructured.
I think adding sub-sections would be much better.
```
### Deploy operator

### Create AWS Secret

### Create Backup CR

### Verify status
```